fineract-branch/src/main/java/org/apache/fineract/organisation/teller/exception/CashierNotFoundException.java

```diff
@@ -40,4 +41,9 @@ public class CashierNotFoundException extends AbstractPlatformResourceNotFoundEx
     public CashierNotFoundException(Long cashierId) {
         super(ERROR_MESSAGE_CODE, DEFAULT_ERROR_MESSAGE, cashierId);
     }
+
+    public CashierNotFoundException(Long id, EmptyResultDataAccessException e) {
+        super("error.msg.staff.id.invalid", "Staff with identifier " + id + " does not exist", id, e);
```

Comment:
This is a cashier not found exception, I think the error message and code should be related to casier and not staff.